Readme.md

```diff
@@ -0,0 +1,20 @@
+## Consumer micro-service
+An example program demonstrating how a consumer service could be written in Go.
+## Setup
+```bash
+git clone github.com/rafaeljesus/srv-consumer
+cd srv-consumer
+make deps
+docker-compose up
```

Comment:
I think we could have a `make build` and than the doccker-compose file would just run the binary [see here](https://github.com/rafaeljesus/composing-go-example/blob/master/Makefile#L13) WDYT?

Reply:
added a dockerfile now and called make build from there